app/models/owner.js

```diff
@@ -88,26 +88,42 @@ export default VcsEntity.extend({
   subscriptionError: reads('accounts.subscriptionError'),
   subscriptions: reads('accounts.subscriptions'),
 
-  subscription: computed(
+  accountSubscriptions: computed(
     'subscriptions.@each.{validTo,owner,isSubscribed,isPending,isIncomplete}',
     'login',
     function () {
       let subscriptions = this.subscriptions || [];
-      const accountSubscriptions = subscriptions.filterBy('owner.login', this.login);
-      const activeAccountSubscriptions = accountSubscriptions.filterBy('isSubscribed');
-      const incompleteAccountSubscriptions = accountSubscriptions.filterBy('isIncomplete');
-      const pendingAccountSubscriptions = accountSubscriptions.filterBy('isPending');
-      if (activeAccountSubscriptions.length > 1 ||
-        pendingAccountSubscriptions.length > 1 ||
-        incompleteAccountSubscriptions.length > 1) {
+      return subscriptions.filterBy('owner.login', this.login);
+    }),
+
+  activeAccountSubscriptions: filterBy('accountSubscriptions', 'isSubscribed'),
+  incompleteAccountSubscriptions: filterBy('accountSubscriptions', 'isIncomplete'),
+  pendingAccountSubscriptions: filterBy('accountSubscriptions', 'isPending'),
+  expiredAccountSubscriptions: filterBy('accountSubscriptions', 'isExpired'),
+  expiredStripeSubscriptions: filterBy('expiredAccountSubscriptions', 'isStripe'),
+
+  expiredStripeSubscription: computed('expiredStripeSubscriptions', function () {
+    if (this.expiredStripeSubscriptions.length > 1) {
+      this.logMultipleSubscriptionsError();
+    }
+    return this.expiredStripeSubscriptions.get('firstObject');
+  }),
+
+  subscription: computed(
+    'accountSubscriptions',
+    'activeAccountSubscriptions',
+    'pendingAccountSubscriptions',
+    'incompleteAccountSubscriptions', function () {
```

Comment:
All array keys should track their array contents with `.[]` or `.@each` or `.length` or `.firstObject` - whatever works best in any particular case.